index.html

```diff
@@ -219,6 +220,36 @@ <h3>Skills</h3>
                 </li>
             </ul>
         </section>
+
+        <section class="contact-me" id="contact-me">
+            <div class="contact-container">
+                <div class="contact-content">
+                    <h2>Contact me</h2>
+                    <p>If you have an application you are interested in developing, a feature that you need built or a project that needs
+                    coding. I’d love to help with it
+                    </p>
+                    <form action="/" method="post">
+                        <ul>
+                            <li>
+                                <input type="text" id="name" placeholder="Enter your name">
+                            </li>
```

Comment:
- [ ] Kindly 😃 It required that the name field accepts up to 30 chars, therefore kindly add the maxlength="30" attribute to the name field.💯👌
 - [ ] Also Please 😊It required that the user shouldn't be able to submit a form when one or more fields are empty, Therefore kindly add the required attribute to the name fields.💯🚀